pkg/registry/apis/secret/reststorage/secure_value_rest.go

```diff
@@ -122,12 +130,34 @@ func (s *SecureValueRest) Create(
 		return nil, err
 	}
 
-	createdSecureValue, err := s.storage.Create(ctx, sv)
-	if err != nil {
-		return nil, fmt.Errorf("failed to create secure value: %w", err)
+	// Assigned inside the transaction callback
+	var object runtime.Object
+
+	if err := s.database.Transaction(ctx, func(ctx context.Context) error {
+		// TODO: return err when secret already exists
```

Comment:
Are these TODOs already addressed? I'm fine with including them here if needed 👍

Reply:
This one is addressed in another PR. The encryption one is not